public/css/common.less

```diff
@@ -403,3 +403,10 @@ div.show-more {
 form[name="form_confirm_removal"] {
   text-align: center;
 }
+
+.state-ball,
+.state-badge {
+  &.state-problem {
```

Comment:
As discussed offline, please change the class to `.state-(reachable|unreachable)`.
Please change the method `RedundencyGroupState::getTextState()` accordingly.

Reply:
I think this only applies for state-ball. In case of state-badges, the objects summary gives the number of objects in problem (down or critical) state. So the css class state-problem is still necessary in case of state-badges.